src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
```

Comment:
Here you can use this too

When you use constructors, classes, or something like that `this` looks to service which call your method
`calculator.reset()` -> this on reset is a calculator
But add is a callback, the function doesn’t have this:
`add()` -> this isn’t exit here

```suggestion
    reset() {
      this.result = 0;

      return this;
    },
```

Reply:
thanks for your advice)